### PR TITLE
Support packages that don't have setup.py

### PR DESCRIPTION
**Pull request by vyasr**
We'll need this feature when packages start migrating to scikit-build-core

### CODE REVIEW

File: etc/conda-activate.sh [hunk 1]
```diff
@@ -70,16 +70,35 @@ export CUGRAPH_BUILD_PATH="$CUGRAPH_ROOT"
...
+    "$CUDF_HOME/python/dask_cudf"
+    "$RAFT_HOME/python/pylibraft"
+    "$RAFT_HOME/python/raft-dask"
+    "$CUML_HOME/python"
+    "$CUGRAPH_HOME/python/cugraph"
+    "$CUGRAPH_HOME/python/pylibcugraph"
+    "$CUSPATIAL_HOME/python/cuspatial"
+)
+
+# Function to a path to PYTHONPATH if it contains a setup.py file Directories
```
Comment (bdice) [inline etc/conda-activate.sh]:
```suggestion
# Function to a path to PYTHONPATH if it contains a setup.py file. Directories
```

**Review by bdice — APPROVED** [review]
Looks good to me. If you can build all of RAPIDS with this change, then feel free to merge.

**Comment by vyasr** [comment]
I can build all the existing packages. I'll wait to merge this until I can test with at least one or two packages after the scikit-build-core conversion.

**Comment by vyasr** [comment]
OK I have this tested with both rmm and cudf in compose now, so I'm confident enough to merge.

**Pull request merged**